index.yaml

```diff
@@ -319,6 +319,19 @@ sources:
     checksum: false
     license: CC-BY-SA-4.0
 
+  opnsense-nmap:
```

Comment:
Are these OPNsense rules, or is that just what they are named. Over on the SID allocations we put these under the org "Aleksi Bovellan", not OPNsense, so I think that should be reflected here to avoid any confusion.

Reply:
Yeah, good question. I'll ask them.

Reply:
https://github.com/aleksibovellan/opnsense-suricata-nmaps/issues/3

There's no direct authorship relation to OPNsense. I'll rename the rules to reflect that.